pkg/cmd/task/list.go

```
@@ -73,23 +76,23 @@ func printTaskDetails(out io.Writer, p cli.Params) error {
 
 	tasks, err := listTaskDetails(cs.Tekton, p.Namespace())
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Failed to list tasks from %s namespace\n", p.Namespace())
-		return err
+		fmt.Fprintln(s.Err, emptyMsg)
```

why are we ignoring this err?

---

addressed